site/src/analytics-integrations.ts

```diff
@@ -20,6 +20,23 @@ import {
 
 const isMobile = matchMedia("(max-width: 600px)").matches;
 
+const IGNORED_DOMAINS = [
+  "analytics",
+  "api",
+  "auth",
+  "config",
+  "device_automation",
+  "frontend",
+  "http",
+  "image",
+  "lovelace",
+  "onboarding",
+  "person",
+  "search",
+  "system_log",
```

Comment:
Suggested: `trace`, as you can't ignore it

```suggestion
  "system_log",
  "trace",
```